### PR TITLE
Rogii v1.2.8 dev

### PR DESCRIPTION
**Pull request by Shkarlatov**

### CODE REVIEW

File: rogii/package.cmake [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    return()
+endif()
+
+add_library(
+    ZLib::library
+    SHARED
+    IMPORTED
+)
+
+set_target_properties(
```
Comment (chak3r) [inline rogii/package.cmake:11]: Actually, I can't remember why we don't need zLib on Linux for console application, but if you're doing CI for both, add the Linux section here. ( as a training;) )

File: rogii/version.cmake [hunk 2]
```diff
@@ -0,0 +1,19 @@
+set(
+    ROGII_ZLIB_VERSION_MAJOR
+    1
+)
+
+set(
```
Comment (chak3r) [inline rogii/version.cmake:6]: copypasted version from rapidjson, i guess

File: rogii/build_common.cmake [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        "${RELEASE_PATH}"
+)
+
+file(
+    COPY
+        ${CMAKE_CURRENT_LIST_DIR}/package.cmake
+    DESTINATION
+        "${ROOT}/${PACKAGE_NAME}"
+)
+
```
Comment (chak3r) [inline rogii/build_common.cmake:123]: Strange empty lines

File: CMakeLists.txt [hunk 4]
```diff
@@ -215,6 +227,15 @@ if(NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL )
         RUNTIME DESTINATION "${INSTALL_BIN_DIR}"
         ARCHIVE DESTINATION "${INSTALL_LIB_DIR}"
         LIBRARY DESTINATION "${INSTALL_LIB_DIR}" )
+    if(MSVC)
+    install(
```
Comment (chak3r) [inline CMakeLists.txt]: need indentation

File: rogii/environment_linux.cmake [hunk 5]
```diff
@@ -0,0 +1,13 @@
+set(
```
Comment (chak3r) [inline rogii/environment_linux.cmake]: REPOSITORY_OS is deprecated

File: rogii/package.cmake (hunk 1 above)
Comment (chak3r) [reply rogii/package.cmake:11]: no need to do

**Review by chak3r — CHANGES_REQUESTED** [review]

**Review by chak3r — APPROVED** [review]

**Review by mr-bit — APPROVED** [review]

**Pull request merged**